solutions/standard/main.tf

```diff
@@ -120,6 +119,65 @@ module "elasticsearch" {
   enable_elser_model            = var.enable_elser_model
 }
 
+#create a service authorization between Secrets Manager and the target service (Elastic Search)
+resource "ibm_iam_authorization_policy" "policy" {
+  count                       = var.skip_es_kms_auth_policy ? 0 : 1
+  depends_on                  = [module.elasticsearch]
+  source_service_name         = "secrets-manager"
+  source_resource_instance_id = local.existing_secrets_manager_instance_guid
+  target_service_name         = "databases-for-elasticsearch"
+  target_resource_instance_id = local.use_existing_db_instance ? data.ibm_database.existing_db_instance[0].guid : module.elasticsearch[0].guid
```

Comment:
This same evaluation is in multiple places (at least here and in output.tf). Perhaps it would be more maintainable as a local.

Reply:
Added this validation,`target_resource_instance_id = local.use_existing_db_instance ? data.ibm_database.existing_db_instance[0].guid : module.elasticsearch[0].guid`  in a local variable, `elasticsearch_guid`